🐛[BUG]2.39.1版本lodash依赖问题

### 🐛   2.39.1版本 lodash.merge 依赖问题 

### 📷 复现步骤

刚刚发布的2.39.1版本 安装依赖之后发布运行直接失败 

2021-05-28 15:15:56.794  * lodash.merge in ./node_modules/@ant-design/pro-form/es/layouts/StepsForm/index.js
2021-05-28 15:15:56.794  
2021-05-28 15:15:56.794  To install it, you can run: npm install --save lodash.merge
2021-05-28 15:15:56.794  Child
2021-05-28 15:15:56.794      
2021-05-28 15:15:56.794      ERROR in ./node_modules/@ant-design/pro-form/es/layouts/StepsForm/index.js
2021-05-28 15:15:56.794      Module not found: Error: Can't resolve 'lodash.merge' in '项目目录/node_modules/@ant-design/pro-form/es/layouts/StepsForm'
2021-05-28 15:15:56.794       @ ./node_modules/@ant-design/pro-form/es/layouts/StepsForm/index.js 241:0-33 339:23-28
2021-05-28 15:15:56.794       @ ./node_modules/@ant-design/pro-form/es/index.js
2021-05-28 15:15:56.794       @ ./node_modules/@ant-design/pro-table/es/utils/cellRenderToFromItem.js
2021-05-28 15:15:56.794       @ ./node_modules/@ant-design/pro-table/es/index.js
2021-05-28 15:15:56.794       @ ./src/pages/heavy-table-demo/TableDemoPage.tsx
2021-05-28 15:15:56.794       @ ./src/pages/heavy-table-demo/index.ts
2021-05-28 15:15:56.794       @ ./src/.umi-production/core/routes.ts
2021-05-28 15:15:56.794       @ ./src/.umi-production/umi.ts
2021-05-28 15:15:56.794       @ multi ./src/.umi-production/umi.ts


### 🏞 期望结果

<!--
描述你原本期望看到的结果
-->

### 💻 复现代码

<!--
提供可复现的代码，仓库，或线上示例
-->

### © 版本信息

- ProComponents 版本: [e.g. 4.0.0]
- umi 版本
- 浏览器环境
- 开发环境 [e.g. mac OS]

### 🚑 其他信息

<!--
如截图等其他信息可以贴在这里
-->

--- a/package.json
+++ b/package.json
@@ -191,7 +191,6 @@
     }
   ],
   "dependencies": {
-    "@umijs/plugin-esbuild": "^1.1.0",
-    "lodash.merge": "^4.6.2"
+    "@umijs/plugin-esbuild": "^1.1.0"
   }
 }

--- a/packages/form/package.json
+++ b/packages/form/package.json
@@ -39,6 +39,7 @@
     "@ant-design/pro-utils": "1.18.1",
     "@umijs/use-params": "^1.0.2",
     "classnames": "^2.2.6",
+    "lodash.merge": "^4.6.2",
     "omit.js": "^2.0.2",
     "rc-resize-observer": "^0.2.3",
     "rc-util": "^5.0.6",

